### DISCUSSION TITLE
Problems with Bluetooth connection to Prime hub

### DISCUSSION THREAD

**Discussion by RPeterslego** [pybricks/support#1696] [Q&A]
I was able to download the firmware and make the connection via bluetooth, at least it says it is paired and running Hello World, I don't get any error messages.  Hello world shows up on the computer screen below the code window, but the hub does nothing.  I can;t get the hub to react to any actions I try, yet it says it is paired?
Any help would be appreciated.
Thanks

**Comment by laurensvalk** [comment]
> Hello world shows up on the computer screen below the code window, but the hub does nothing

That sounds good. Which example are you running? The "hello world" example does nothing else but print that message. You can add additional blocks to make a motor move, for example:

![image](https://github.com/pybricks/support/assets/12326241/568af9a8-dbca-4fa6-9b29-c0f1e7467876)

**Reply by RPeterslego** [reply]
Thanks so much for responding! That worked!  I didn't have the set up step in there!  My goal here is to drive the hub with the remote controller from Lego.  I am guessing I need a set up block for the remote controller as well to make it work as well?

**Comment by RPeterslego** [comment]
So could use some help here as well.  I have the remote in the set up block, but I seem to be stuck in a loop.  When I include the remote input, it says I need to set it up, but it is already set up?

![image](https://github.com/pybricks/support/assets/174194130/005cd82c-0378-4a55-b593-1301aaf6db2b)

**Reply by laurensvalk** [reply]
Here is an example. See also: https://github.com/orgs/pybricks/discussions/1420

![image](https://github.com/pybricks/support/assets/12326241/81362cb6-cad1-4fb8-ad36-1ee65d845613)

**Reply by RPeterslego** [reply]
Thank you again so much. It really wasn't very intuitive....  Thanks for the link as well, very helpful!  You guys are great!
